28-04/task_2/task_2.rb

```diff
@@ -0,0 +1,8 @@
+File.open("palindrom.txt").each do |line|
+  palindrome = line.strip.downcase.delete("/\s\.\,/")
+    if palindrome == palindrome.reverse
+      puts "True"
+    else
+      puts "False"
+    end
+end
```

Comment:
You need to add a newline at the end of each ruby file. you can check the history of this requirement here https://thoughtbot.com/blog/no-newline-at-end-of-file